libpod/oci_conmon_linux.go

```diff
@@ -1748,3 +1755,187 @@ func httpAttachNonTerminalCopy(container *net.UnixConn, http *bufio.ReadWriter,
 		}
 	}
 }
+
+// GetLimits converts spec resource limits to cgroup consumable limits
+func Getlimits(resource *spec.LinuxResources) (runcconfig.Resources, error) {
+	if resource == nil {
```

Comment:
the code uses `GetLimits` but the function is `Getlimits`, let's use `GetLimits`